### ISSUE TITLE
y*a*ml file extension raise un user friendly message.

### ISSUE THREAD

**Issue by hiroyuki-sato** [embulk/embulk#313]
If I use file extension except ``yml`` or yml.liquid, 
the following exception raised.

But It's not user friendly, Should output following message which 
is raise on runner.rb: line.120.

```
Unsupported file extension. Supported file extensions are .yml and .yml.liquid: yaml
```

Test procedure.

```
embulk example test1
cp test1/example.yml test1/example.yaml 
embulk guess test1/example.yaml -o config.yml
```

```
embulk guess test1/example.yaml -o config.yml
2015-09-30 10:00:34.089 +0900: Embulk v0.7.4
exception class/object expected
    /path/to/home/.embulk/bin/embulk!/embulk/runner.rb:120:in `read_config'
    /path/to/home/.embulk/bin/embulk!/embulk/runner.rb:12:in `guess'
    /path/to/home/.embulk/bin/embulk!/embulk/command/embulk_run.rb:274:in `run'
    /path/to/home/.embulk/bin/embulk!/embulk/command/embulk_main.rb:2:in `<top>'
    org/jruby/RubyKernel.java:940:in `require'
    uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:1:in `(root)'
    file:/path/to/home/.embulk/bin/embulk!/embulk/command/embulk_bundle.rb:55:in `<top>'

Error: exception class/object expected
```

**Comment by frsyuki** [comment]
Good point.

**Comment by frsyuki** [comment]
Fixed at 69d7ac2a5cdd38baa7f6fbd01e9163ef9a9b0c4e. Thank you for reporting!

**Comment by hiroyuki-sato** [comment]
Thanks!
